src/Workspaces/CSharp/Portable/Recommendations/CSharpRecommendationServiceRunner.cs

```diff
@@ -500,7 +505,7 @@ private ImmutableArray<ISymbol> GetUnnamedSymbols(ExpressionSyntax originalExpre
             using var _ = ArrayBuilder<ISymbol>.GetInstance(out var symbols);
```

Comment:
NIT: Could be moved after the conditional `return`, if container is null (Line 510).